tasks/prepare-binaries.js

```diff
@@ -71,28 +71,38 @@ function copyAssets (platforms) {
 
 async function archive (platforms) {
   console.log('Zipping...')
-  await Promise.all(Object.keys(platforms).map(async (platform) => {
+
+  const promises = Object.keys(platforms).map(async (platform) => {
```

Comment:
I guess we could use the following:

```js
  return Promise.all(Object.keys(platforms).map(async (platform) => { /* ... */
```

We should also probably update `getBrowsers` to be consistent:

```diff
- await Promise.all(Object.entries(platforms).map(async ([name, platform], index) => {
+ return Promise.all(Object.entries(platforms).map(async ([name, platform], index) => {
```

 .

Reply:
Changed `getBrowsers()` to return the promise. Moved the log output about downloaded browsers to `main()`.